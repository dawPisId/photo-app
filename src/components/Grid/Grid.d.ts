interface GridInterface{
    children:JSX.Element
}