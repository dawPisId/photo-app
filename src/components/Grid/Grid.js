import {Item, Layout} from "./Grid.styled"

import React from "react";

const Grid = ({children}) => (
  <Layout>
  </Layout>

);

export default Grid;

