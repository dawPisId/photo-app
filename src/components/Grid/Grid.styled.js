import styled from "styled-components";

export const Layout = styled.div`
  display: grid;
  grid-template-columns: 1fr 1fr 1fr 1fr;
  grid-gap: 1rem;
  margin-bottom: 2rem;
`;
