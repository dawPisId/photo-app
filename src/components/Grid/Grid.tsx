import { Layout } from "./Grid.styled";
import React from "react";

const Grid = ({ children }:any) => <Layout>{children}</Layout>;

export default Grid;
