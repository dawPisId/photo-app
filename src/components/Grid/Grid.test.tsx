import Grid from "./Grid";
import React from "react";
import renderer from "react-test-renderer";

it("renders correctly", () => {
  const tree = renderer
    .create(
      <Grid>
        <div>One</div>
        <div>Two</div>
        <div>Three</div>
        <div>Four</div>
      </Grid>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
