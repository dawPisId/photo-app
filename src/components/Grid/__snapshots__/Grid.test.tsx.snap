// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sc-bdnxRM AWwxL"
>
  <div>
    One
  </div>
  <div>
    Two
  </div>
  <div>
    Three
  </div>
  <div>
    Four
  </div>
</div>
`;
